ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline.yml

```diff
@@ -261,9 +267,18 @@ spec:
           value: *bundleIndexImage
         - name: bundle_image
           value: *bundleImage
+        - name: namespace
+          value: "$(params.preflight_namespace)"
+        - name: service_account
+          value: "$(params.preflight_service_account)"
       workspaces:
+        - name: output
```

Comment:
nitpick: we could store that in some other workspace, than the one which already contains the repository